pkg/odo/cli/init/init.go

```diff
@@ -125,6 +125,16 @@ func (o *InitOptions) Run() (err error) {
 		}
 	}()
 
+	isEmptyDir, err := location.DirIsEmpty(o.clientset.FS, o.contextDir)
+	if err != nil {
+		return err
+	}
+	if isEmptyDir && len(o.flags) == 0 {
+		fmt.Println("The current directory is empty.\nOdo will help with you starting a new project.")
+	} else if len(o.flags) == 0 {
+		fmt.Println("The current directory already contains source code.\nOdo will try to autodetect language and project type to select best suited Devfile for your project.")
```

Comment:
odo should always be lower case :) never capitalized

Reply:
Comment below : https://github.com/redhat-developer/odo/pull/5514#discussion_r820491142